upgrade/1.5.3/CSM-With-Other-Products.md

```diff
@@ -0,0 +1,282 @@
+# CSM Only Upgrade on a System with Other Products
+
+* [Requirements](#requirements)
+* [Overview](#overview)
+* [Steps](#steps)
+    * [Using `sat bootprep` with IUF generated input files](#using-sat-bootprep-with-iuf-generated-input-files)
+* [Return to CSM `1.5.3` patch](#return-to-csm-153-patch)
+
+This page provides guidance for systems with products installed that are performing an upgrade from a CSM `v1.5.X` release to the CSM `v1.5.3` release.
+
+The [`v1.5.3` upgrade page](README.md) will refer to this page during [Update NCN images](README.md#update-ncn-images).
+
+**If exclusively CSM is installed on the system, return to [Update NCN images](README.md#update-ncn-images) and choose option 2.**
```

Comment:
Confused by the wording here, should it be "If CSM is exclusively installed"?

Reply:
If so, we'll want to edit it on all of the other patch upgrade pages. Most of this procedure was copied from https://github.com/Cray-HPE/docs-csm/blob/release/1.5/upgrade/1.5.2/README.md

I'm fine either way.

Reply:
Changed